docs/platforms/javascript/guides/nuxt/install/dynamic-import.mdx

```diff
@@ -6,37 +6,17 @@ description: "Learn about how the Nuxt SDK leverages dynamic input() in the buil
 
 ## Understanding the `import()` expression
 
-The `import()` expression (also called "dynamic import") allows conditional and flexible module loading in ESM.
-For the Sentry Nuxt SDK, it provides an approach to initialize server-side configuration before application startup.
+<Alert level='warning'>
+  This installation method comes with restrictions when using more recent versions of Nuxt/Nitro.
```

Comment:
```suggestion
  This installation method does not work with more recent versions of Nuxt/Nitro.
```

is this not more correct? 😅

Reply:
It can work sometimes 😄 I had it working with newer versions, but only under certain conditions.

But yeah, probably it's better to just say it's not working to be more precise.